packages/synapse-react-client/src/components/dataaccess/AccessHistoryDashboard.tsx

```diff
@@ -16,6 +16,19 @@ import UserOrTeamBadge from '../UserOrTeamBadge/UserOrTeamBadge'
 import UserSearchBoxV2 from '../UserSearchBox/UserSearchBoxV2'
 import AccessRequirementSearchBox from './AccessRequirementSearchBox/AccessRequirementSearchBox'
 import { useGetCurrentUserBundle } from '../../synapse-queries'
+import SearchState from '../../assets/icons/SearchState'
+
+const SearchStartComponent = () => {
+  return (
+    <div className="text-center">
+      <SearchState className="SearchPlaceholderImage" height="167px" />
+      <Typography variant="body1Italic">
+        Select a user using the search field above to view their data access
+        history
+      </Typography>
+    </div>
+  )
+}
```

Comment:
Moved this component definition outside of its parent to prevent unnecessary re-rendering and renamed to reflect which search image is used. Otherwise, only changed to use the SearchState component instead of the S3 image --

main | feature 
:---:|:---:
<img width="1178" alt="SWC-6719_accessHistoryDashboard_main" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/48059562-1eb7-4d6f-a84a-0d4f2f919b20"> | <img width="1178" alt="SWC-6719_accessHistoryDashboard_feature" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/a2a5a24a-402a-45ed-9b7c-de7f3b4e3d33">